{
  Exception e=new TestException();
  Utils.returnFirstExceptionIfInListOrThrowStandardExceptionOrCause(new Class[]{},e);
}
